cluster-autoscaler/processors/capacitybuffer/pod_list_processor.go

```diff
@@ -190,7 +195,16 @@ func withCapacityBufferFakePodAnnotation(pod *apiv1.Pod) *apiv1.Pod {
 	return pod
 }
 
-func isFakeCapacityBuffersPod(pod *apiv1.Pod) bool {
+func withSafeToEvictAnnotation(pod *apiv1.Pod) *apiv1.Pod {
+	if pod.Annotations == nil {
+		pod.Annotations = make(map[string]string, 1)
+	}
+	pod.Annotations[drain.PodSafeToEvictKey] = "true"
```

Comment:
Because we're passing in a static `true` for this option, this represents a change. We're O.K. that given that this functionality hasn't yet been released?